Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -100,14 +100,23 @@ def customisePostEra_Run2_2018_pp_on_AA_express_trackingOnly(process):
     return process
 
 # 2021 equivalents
+def customise2021(process):
```

Comment:
```suggestion
def _hcalCustomsRun3(process):
```
to keep some consistency with what is done for other modifications (`_customiseHcal_Run3` may be fine as well)
- add a leading underscore to avoid imports by simple inclusion of `RecoTLR *`
- be specific that this is for hcal
- I think that for now we better say Run3 than a specific year